src/constants/index.js

```diff
@@ -64,17 +64,19 @@ export const distributionMapper = {
   'rhel-84': 'RHEL 8.4',
   'rhel-85': 'RHEL 8.5',
   'rhel-86': 'RHEL 8.6',
+  'rhel-87': 'RHEL 8.7',
   'rhel-90': 'RHEL 9.0',
 };
 
 export const releaseMapper = {
   'rhel-90': 'Red Hat Enterprise Linux (RHEL) 9.0',
+  'rhel-87': 'Red Hat Enterprise Linux (RHEL) 8.7',
   'rhel-86': 'Red Hat Enterprise Linux (RHEL) 8.6',
   'rhel-85': 'Red Hat Enterprise Linux (RHEL) 8.5',
   'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
 };
 
-export const supportedReleases = ['rhel-84', 'rhel-85'];
+export const supportedReleases = ['rhel-84', 'rhel-85', 'rhel-87'];
```

Comment:
What does `supportedReleases` mean? We should be supporting both `rhel-86` and `rhel-90` right now. Also, if `temporaryReleases` below is a way to track releases that we're not fully supporting yet, then we could add `reel-87` there until GA?

Reply:
I believe supportedReleases does mean releases we are officially supporting and temporyReleases are ones we are testing and will officially support soon.
@ldjebran Is that correct?

Reply:
@omaciel per code read, we use the temporary and default to enable or disable the versions based on the feature flag.

Reply:
I believe we can move all to default and add only 91 to temporary given is not in ga